### PR TITLE
(test): Add test for `patient-table.component.tsx`

### PR DESCRIPTION
**Pull request by ayush-AI**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

In this PR, I have added test for `patient-table.component.tsx`

## Screenshots

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

**Comment by ayush-AI** [comment]
cc/ @Piumal1999 @anjula-sack @jayasanka-sack

File: packages/esm-patient-list-app/src/patient-table/patient-table.component.tsx [hunk 1]
```diff
@@ -101,7 +109,7 @@ const PatientTable: React.FC<PatientTableProps> = ({
               id="patient-list-search"
               placeholder={search.placeHolder}
               labelText=""
-              size={isDesktop(layout) ? 'sm' : 'xl'}
+              size={isDesktop(layout) ? 'sm' : 'lg'}
```
Comment (Piumal1999) [inline packages/esm-patient-list-app/src/patient-table/patient-table.component.tsx:112]: Hi @ayush-AI, Is this change required for the tests?

Comment (ayush-AI) [reply packages/esm-patient-list-app/src/patient-table/patient-table.component.tsx:112]:
```
 console.error
      Warning: Failed prop type: Invalid prop `size` of value `xl` supplied to `ForwardRef(Search)`, expected one of ["sm","md","lg"].
          at Search (/Users/ayushmishra/Desktop/openmrs/openmrs-esm-patient-management/node_modules/@carbon/react/lib/components/Search/next/Search.js:38:32)
          at patients (/Users/ayushmishra/Desktop/openmrs/openmrs-esm-patient-management/packages/esm-patient-list-app/src/patient-table/patient-table.component.tsx:57:3)
          at div
          at section
          at main
          at PatientListDetailComponent (/Users/ayushmishra/Desktop/openmrs/openmrs-esm-patient-management/packages/esm-patient-list-app/src/patient-list-detail/patient-list-detail.component.tsx:23:27)

      106 |         <div>
      107 |           <Layer>
    > 108 |             <Search
          |             ^
      109 |               id="patient-list-search"
      110 |               placeholder={search.placeHolder}
      111 |               labelText=""

      at printWarning (node_modules/react/cjs/react.development.js:209:30)
      at error (node_modules/react/cjs/react.development.js:183:7)
      at checkPropTypes (node_modules/react/cjs/react.development.js:2074:11)
      at validatePropTypes (node_modules/react/cjs/react.development.js:2275:7)
      at Object.createElementWithValidation [as createElement] (node_modules/react/cjs/react.development.js:2379:5)
      at PatientTable (packages/esm-patient-list-app/src/patient-table/patient-table.component.tsx:108:13)
      at renderWithHooks (node_modules/react-dom/cjs/react-dom.development.js:16305:18)
      at mountIndeterminateComponent (node_modules/react-dom/cjs/react-dom.development.js:20074:13)
   ```
   
   
 I was getting this error that's why I changed.

**Review by Piumal1999 — APPROVED** [review]
